rpc/src/client/transport/router.rs

```diff
@@ -17,36 +19,54 @@ pub struct SubscriptionRouter {
     // their result channels. Used for publishing events relating to a specific
     // query.
     subscriptions: HashMap<String, HashMap<String, SubscriptionTx>>,
+
+    // A map of subscription ids to their queries
+    sub_to_query: HashMap<String, String>,
 }
 
 impl SubscriptionRouter {
+    pub fn publish_error(&mut self, id: &str, err: Error) -> PublishResult {
```

Comment:
Not sure why Clippy is complaining as this method is used in `rpc/src/client/transport/websocket.rs` at line 688.

Reply:
Might be due to the features enabled in CI when running the clippy check.